### PR TITLE
Backport the fix for xenial package cluster initialization

### PR DESCRIPTION
**Pull request by lazypower**
This backports the bits which remove the single-node initialization (this only affects first deploy, from the xenial package).

### CODE REVIEW

File: reactive/etcd.py [hunk 1]
```diff
@@ -129,6 +131,14 @@ def install_etcd():
...
+            # Stop the service and remove the defaults
+            # I hate that I have to do this. Sorry short-lived local data #RIP
+            # State control is to prevent upgrade-charm from nuking cluster
+            # data.
+            if not is_state('etcd.package.adjusted'):
+                host.service('stop', 'etcd')
+                rmtree('/var/lib/etcd/default')
+                set_state('etcd.package.adjusted')
             set_state('etcd.installed')
             return
```
Comment (mbruzek) [inline reactive/etcd.py:143]: Don't you need to start the etcd service before returning from the "install_etcd" method here? The service was running, I don't know what would start it again.

**Comment by mbruzek** [comment]
I hate to delete data like that but I know of no other way to fix it.

File: reactive/etcd.py (hunk 1 above)
Comment (lazypower) [inline reactive/etcd.py:143]: nah, here's why

https://github.com/chuckbutler/layer-etcd/blob/313626c724318c444ad7d71904f7aa22aec72a94/reactive/etcd.py#L216

**Comment by mbruzek** [comment]
I knew the restart was somewhere. I just could not find it. +1

**Comment by lazypower** [comment]
Yep i totally get that. If we re-start before we have inserted our configuration data for the static-cluster style of turn-up, we wind up in the exact same boat as we started in using the xenial package which is targted at a single-node localhost style deployment.

Glad you were keeping me honest sir 👍

**Pull request merged**